src/main/java/core/basesyntax/strategy/impl/ReturnOperationImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationHandler;
+
+public class ReturnOperationImpl implements OperationHandler {
+    @Override
+    public void applyOperation(FruitTransaction transaction) {
+        if (transaction.getQuantity() <= MIN_QUANTITY) {
+            throw new RuntimeException("You must return 1 or more items. But was: "
+                    + transaction.getQuantity());
+        }
+        Storage.reports.put(transaction.getFruitName(),
+                transaction.getQuantity() + getCurrentBalance(transaction));
```

Comment:
The method 'getCurrentBalance()' is being used but is not defined or imported in this class. Please ensure that you have the method implemented or imported from the correct location.